.vib/moodle/cypress/cypress/integration/moodle_spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+it('adds new course', () => {
```

Comment:
I would change this for something more descriptive like:

```suggestion
it('allows adding a new course with image', () => {
```